vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -992,16 +1014,17 @@ def _prepare_prompt(
         self.req_ids2reserved_blocks[req_id] = n_reserved_blocks
```

Comment:
I was a bit confused reading the code because I assumed this dictionary mapped the request id to the list of block IDs that were reserved, and couldn't understand how that interacted with the KV cache manager. Now I understand it is just the *number* of reserved blocks. Maybe we could change the name to something like `req_ids2num_reserved_blocks` or something (as a follow-up).